frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield.jsx

```diff
@@ -320,6 +360,29 @@ export default class ExpressionEditorTextfield extends React.Component {
       <EditorContainer isFocused={isFocused}>
         <EditorEqualsSign>=</EditorEqualsSign>
         <AceEditor
+          commands={[
+            {
+              name: "arrowDown",
+              bindKey: { win: "Down", mac: "Down" },
+              exec: () => {
+                this.handleArrowDown();
+              },
+            },
+            {
+              name: "arrowUp",
+              bindKey: { win: "Up", mac: "Up" },
+              exec: () => {
+                this.handleArrowUp();
+              },
+            },
+            {
+              name: "enter",
```

Comment:
Is `Tab` (same function as `Enter`) also handled somewhere?

Reply:
It will soon, thanks for the reminder.

Reply:
For the follow-up/next PR: `Tab` only behaves like `Enter` **when** there is at least one suggestion.
If there is no suggestion/popover, then it should still cycle focus/**unblur**, as usual.

Reply:
Hmm right, will add that o/